tests/features/menus/feature_set.feature

```diff
@@ -75,7 +75,7 @@ Feature: Feature set menu
     Then I should not see the text "F.A.Q"
     Then I should not see the text "Press Release"
     Then I should not see the text "Site activity"
-    Then I should not see the text "Maxlength"
+#    Then I should not see the text "Maxlength"
```

Comment:
Hi,
This seems unnecessary for me as this test is not about translations.
The PR should contains  changes for the issue only.
There is no reason that the text 'Maxlength' appears for this user because of your patch.
Seems like a CI bug.